app/src/main/java/org/mifos/mobile/ui/fragments/SavingsAccountApplicationFragment.kt

```diff
@@ -122,7 +122,12 @@ class SavingsAccountApplicationFragment : BaseFragment(), SavingsAccountApplicat
     private fun submitSavingsAccountApplication() {
         val payload = SavingsAccountApplicationPayload()
         payload.clientId = template?.clientId
+        if(productIdList.indexOf(productIdField!!.text.toString())!=-1)
```

Comment:
- Use safe call operator `(?.)`
- Add brackets for `if`
- Add space between ` != ` and` -1` ( Or just press CTRL+ALT+L on Windows or Command+Option+L on Mac. The android studio will reformat all the code for you. I would recommend following this best practice in your daily coding)

Reply:
@jawidMuhammadi i have done the changes, kindly review them